### PR TITLE
✅🏗 Reduce SauceLabs batch size and remove flaky browsers to unblock master

### PR DESCRIPTION
**Pull request by danielrozenberg**

### CODE REVIEW

**Comment by danielrozenberg** [comment]
Done

**Comment by estherkim** [comment]
I'm seeing this error with `amp-bind` in Chrome 71 that I've seen in other recent failing integration tests. Are we sure builds are failing due to flakiness?

`✗ Error: Timeout waiting for All AMP components are laid out
          at poll (home/travis/build/ampproject/amphtml/testing/iframe.js:423:18)`

**Comment by danielrozenberg** [comment]
Flaky or not, these tests are blocking master. They should be fixed (either the test or the code being tested) but for now we should also unblock master. I'm about to add a bunch of `.skip()`s to this PR, then after merging I'll create issues to unskip them

File: test/integration/test-errors.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ t.run('error page', function() {
   });
 
   it.configure().skipFirefox().skipEdge()
-      .run('should show the body in error test', () => {
+      .skip('should show the body in error test', () => {
```
Comment (rsimha) [inline test/integration/test-errors.js]: All `skip`s should ideally have a TODO, but let's not block this PR purely for this.

**Pull request merged**